diagnostics/TC_MSE/TC_MSE.html

```diff
@@ -0,0 +1,27 @@
+<title>TC MSE Variance Budget</title>
+<img src="../mdtf_diag_banner.png">
+
+<p>
+This POD reads in model track data to create 10 degree by 10 degree snapshots around a tropical cyclone (TC) center. The snapshots are made for all storms between 
+the years of 1979 and 1983 and the column-integrated moist static energy (MSE) budget diabatic terms are calculated for in various ways (azimuthal mean, box-averaged
```

Comment:
Does the period have to span 1979--1983, or can any (5-yr?) period be analyzed?

Reply:
Apologies, the period does not specifically have to be 1979-1983. That was just the model data we had to test the POD through the framework. A user can run this POD for any time period necessary. I will adjust the above code to account for this which should hopefully be done by the end of the day today. I will commit all the changes throughout the day. Thank you!